README.md

```diff
@@ -1,13 +1,20 @@
 # vitasdk code samples
 
-Every directory contains a single sample. To build all samples, add the toolchain `bin/` directory to your `$PATH`, then run `./build.sh`. To clean all samples, run `./clean.sh`. To build a single sample, `cd` to its directory and type `make`.
+## Prerequists
 
-Always use the `*_debug.bat` (e.g. `run_homebrew_unity_debug.bat`) launcher to get debug output from Vita.
+In order to build a vita sample, you need to add the toolchain `bin/` directory to your `$PATH`.
 
-## List of samples
+## Building
 
-* `hello_world`: A minimal Makefile-based C project
-* `hello_cpp_world`: A minimal Makefile-based C++ project
+Every directory contains a single sample. `cd` to its directory and use the `make` command to build it.
+
+## Building everything
+
+Use the following command to build every samples
+
+```
+for f in */Makefile; do make -C ${f%/*} all; done
```

Comment:
As said on https://github.com/vitasdk/samples/pull/4

> Basically, when this repository will have dozens of samples (I hope it will),  I seriously doubt that people will build all of them. They will simply go the the sample they are interested in an build-install-run it.